### ISSUE TITLE
own data

### ISSUE THREAD

**Issue by shichao1232** [3DTopia/LGM#72]
How can I convert my images to a format like gobjeverse?

**Comment by xiyichen** [comment]
I was able to do inference with my own data by converting the camera parameters to opengl convention. I also found it important to crop the images so that the objects are centered, and to resize the images to 256x256.

**Comment by shichao1232** [comment]
> I was able to do inference with my own data by converting the camera parameters to opengl convention. I also found it important to crop the images so that the objects are centered, and to resize the images to 256x256.
I don't konw how to do this ,can you teach me? how can i get the  camera parameters? Is it okay to use colmap?

**Comment by xiyichen** [comment]
Yes, you can use colmap, but it uses opencv convention. I found this function https://kit.kiui.moe/_modules/kiui/cam/#convert helpful for camera coordinates convertion.
